intake_xarray/opendap.py

```diff
@@ -37,6 +36,7 @@ def _get_session(self):
             check_url=self.urlpath)
 
     def _open_dataset(self):
+        import xarray as xr
```

Comment:
Should they all just be like this so that we just raise the default import error, or do we want something special?